pages/c/_cluster/apps/install.vue

```diff
@@ -492,8 +494,15 @@ export default {
             nue.disabled = true;
           }
         }
+        if (!this.showPrerelease) {
+          const isPre = !!(version.version || '').match(/.[0-9]+(-[0-9A-Za-z-]+)/);
```

Comment:
This and the other one in charts.vue should be combined so the logic for "is <str> a prerelease" is in only one place (utils/version.js would be a good one).

Metadata (after the `+`) can contain numbers and hyphens too so this doesn't quite work (e.g. `v1.18.6+k3s1-1` will match but is not a prerelease).  You'd have to split off the `+` section first then see if the remainder has a `-`.

But.. you said the library only looks for alpha/beta/etc, which library is that?  The `semver` we have already loaded seems to work?  It's a whole formal generated grammar even, and the words alpha/beta don't appear in it, outside of comments:

```javascript
> const x = require('semver');
> semver.parse('v1.2.3-foo1+k3s2-2');
SemVer {
  options: { loose: false, includePrerelease: false },
  loose: false,
  raw: 'v1.2.3-foo1+k3s2-2',
  major: 1,
  minor: 2,
  patch: 3,
  prerelease: [ 'foo1' ],
  build: [ 'k3s2-2' ],
  version: '1.2.3-foo1'
}
> semver.prerelase('v1.2.3-foo1+bar2-3');
[ 'foo1' ]
> semver.prerelease('v1.2.3');
null
```